### PR TITLE
[SWL-4374] Add an enum for FEC configuration state

### PR DESCRIPTION
**Pull request by nick-bsn**
Reviewer: @kenchiang @wilmo119 

* Add an enum as reviewed for the FEC configuration state

### CODE REVIEW

File: openflow_input/bsn_forward_error_correction [hunk 1]
```diff
@@ -28,11 +28,17 @@
 #version 5
 #version 6
 
+enum ofp_bsn_fec_config_state(wire_type=uint16_t) {
+    OFP_BSN_FEC_CONFIG_STATE_DEFAULT = 0,
```
Comment (kenchiang) [inline openflow_input/bsn_forward_error_correction]: let's add a comment here to describe what "DEFAULT" actually means

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/160/

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-586-jenkins-loxigen_pull_req_artifacts-148

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/148/

**Comment by nick-bsn** [comment]
@kenchiang Pushed new change and please take a look again. Thanks,

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-586-jenkins-loxigen_pull_req_artifacts-149

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/149/

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/161/

**Comment by nick-bsn** [comment]
@kenchiang Please take a look again. Thanks,

**Comment by kenchiang** [comment]
ACK
thanks for taking care of this

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-586-jenkins-loxigen_pull_req_artifacts-150

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/150/

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/162/

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  https://jenkins.bigswitch.com/job/loxigen-abat-automerge/1158//console

**Comment by bsn-abat** [comment]
ABAT: AUTO-REJECT:  Your change failed at real push: Merge error: `[rejected] (fetch first)
` 


INSTRUCTIONS: Please fix your problem by

1) Reviewing the logs below and `git commit`ing a fix.

2) Use `git push` to add new commits to this same remote branch/pull request.

3) Get another code review approval from your co-worker indicated by a line of "ACK" in comment, or add another comment yourself with a line of "R_E_T_R_Y (Remove the underscores)". That will cause abat to try to merge your code again.



https://jenkins.bigswitch.com/job/loxigen-abat-automerge/1158//console
/cc

**Pull request merged**